### ISSUE TITLE
Trim function of AHMI not working as expected

### ISSUE THREAD

**Issue by philippeller** [bat/BAT.jl#97]
The function defined here https://github.com/bat/BAT.jl/blob/master/src/integration/ahmi/util.jl#L2 is supposed to cut out the central 68% of the integral estimates, i.e. reject outliers. If less than 3 estimates are provided it does not make a cut.

Test:

- `BAT._trim( [-1., 0, 1, 2, 3]) = ([1, 5], [2, 3, 4])` looks fine,
- However `BAT._trim([-10., 0, 1, 2, 3]) = ([1, 2, 5], [3, 4])` while expecting `([1, 5], [2, 3, 4])`
- and `BAT._trim([0., 0., 1, 2, 3]) = ([1, 5], [2, 3, 4])` while expecting  `([5], [1, 2, 3, 4])`

I propose the following as a replacement:
```julia
function _trim(a)
    
    if length(a) < 3
        return collect(range(1,stop=length(a)))
    end
        
    lower = quantile(a, 0.15865)
    upper = quantile(a, 0.84135)
    indices = findall(x-> x >= lower && x <= upper, a)
    return (setdiff(range(1,stop=length(a)), indices), indices)
end```

**Comment by oschulz** [comment]
Sounds good - want to make a PR? Maybe also include the tests you proposed?

**Comment by oschulz** [comment]
Or maybe @VasylHafych?

**Comment by philippeller** [comment]
If @VasylHafych can help me with getting set up in 'dev' mode i can learn how to do that and make a PR

**Comment by oschulz** [comment]
Fixed in #97
